### PR TITLE
[rllib] support running older version tensorflow(version < 1.5.0)

### PR DESCRIPTION
**Pull request by adoda**
support run rllib's examples with older version tensorflow(version < 1.5.0)

## What do these changes do?
support run the rllib's examples with tensorflow (version < 1.5.0)

- tf.boolean_mask only support tf.bool when tf version < 1.5.0

- tf.variable_scope support auxiliary_name_scope when tf version >= 1.5.0

## Related issue number
#3570

### CODE REVIEW

File: python/ray/rllib/examples/multiagent_cartpole.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import argparse
 import gym
 import random
+import distutils.version
```
Comment (richardliaw) [inline python/ray/rllib/examples/multiagent_cartpole.py]: I think the examples which are public facing don't need to support older versions of TF.

File: python/ray/tune/logger.py [hunk 2]
```diff
@@ -18,8 +19,11 @@
 
 try:
     import tensorflow as tf
+    use_tf150_api = (distutils.version.LooseVersion(tf.VERSION) >=
+                     distutils.version.LooseVersion("1.5.0"))
 except ImportError:
     tf = None
+    use_tf150_api=True
```
Comment (richardliaw) [inline python/ray/tune/logger.py]:
```suggestion
     use_tf150_api = True
```

**Review by richardliaw — COMMENTED** [review]
We shouldn't clutter the example but the rest of changes look fine to me

File: python/ray/rllib/examples/multiagent_cartpole.py (hunk 1 above)
Comment (adoda) [reply python/ray/rllib/examples/multiagent_cartpole.py]: done

File: python/ray/tune/logger.py (hunk 2 above)
Comment (adoda) [reply python/ray/tune/logger.py]: done

**Comment by adoda** [comment]
@richardliaw updated

**Comment by adoda** [comment]
unrelated error @richardliaw

**Review by richardliaw — APPROVED** [review]

**Pull request merged**